samples/go/Makefile

```
@@ -0,0 +1,24 @@
+local/clean:
+	rm -rf bin
+
+local/build:
+	go build -o bin/leadIntegration;
+
+local/test:
+	go test
+
+local/run:
+	go run main.go
+
+local/setup:
+	go get
```

as you're using go modules, this could be

```suggestion
	go mod vendor
```

---

Updated